### DISCUSSION TITLE
SQL migration and api tokens

### DISCUSSION THREAD

**Discussion by donPuerto** [adonisjs/core#2319] [General]
I have the same problem with  [Problem with migration (MySQL)]#1675](https://github.com/adonisjs/core/discussions/1675).
  /**
       * "useTz: true" utilizes timezone option in PostgreSQL and MSSQL
 */
      table.timestamp('expires_at', { useTz: true }).nullable()
      table.timestamp('created_at', { useTz: true }).notNullable()
into

	table.timestamps(true, true)
It should work.

It works on the migration however ``table.timestamps(true, true)` will only create `created_at` and `updated_at` not the expires_at column. See below image
![image](https://user-images.githubusercontent.com/9329005/109417849-4c158700-7a00-11eb-9047-8c25d88d7a41.png)

**So, the result "ER_BAD_FIELD_ERROR: Unknown column 'expires_at' in 'field list'"**
![image](https://user-images.githubusercontent.com/9329005/109417872-60f21a80-7a00-11eb-9595-d6fc264b4d9f.png)

How do we resolve the issue?
To have `created_at` and `expires_at`

**Comment by devwalex** [comment]
Have you been able to resolve the issue? because I am also having the same issue. @donPuerto

**Reply by cabe7733** [reply]
https://github.com/adonisjs/core/issues/2281
